### PR TITLE
Replace padding with leftAndRightPadding & topAndBottomPadding in TileGroupMap

### PR DESCRIPTION
**Pull request by lishaoxia1985**
Some edit :
1. Replace padding with leftAndRightPadding & topAndBottomPadding in class TileGroupMap, it much better for portrait
2. Edit MainMenuScreen
YueR

### CODE REVIEW

File: core/src/com/unciv/MainMenuScreen.kt [hunk 1]
```diff
@@ -55,21 +48,14 @@ class MainMenuScreen: CameraStageBaseScreen() {
...
-                                touchable = Touchable.disabled
-                                setScale(1f)
-                                center(this@MainMenuScreen.stage)
-                                layout()
-                            }
-                            backgroundTable.add(mapHolder).size(stage.width, stage.height)
-                        },
-                        Actions.fadeIn(0.3f)
-                ))
-
```
Comment (yairm210) [inline core/src/com/unciv/MainMenuScreen.kt:72]: Why did you remove the fade in?

Comment (lishaoxia1985) [reply core/src/com/unciv/MainMenuScreen.kt]: @yairm210 Now it's OK, I reedit this.

File: core/src/com/unciv/ui/utils/ImageGetter.kt [hunk 2]
```diff
@@ -257,7 +257,7 @@ object ImageGetter {
     fun getTriangle() = getImage("OtherIcons/Triangle")
 
     fun getBackground(color: Color): Drawable {
-        val drawable = getDrawable("OtherIcons/TableBackground")
+        val drawable = getDrawable("")
```
Comment (yairm210) [inline core/src/com/unciv/ui/utils/ImageGetter.kt:260]: Was this intentional?

File: core/src/com/unciv/ui/worldscreen/WorldMapHolder.kt [hunk 3]
```diff
@@ -87,8 +87,7 @@ class WorldMapHolder(internal val worldScreen: WorldScreen, internal val tileMap
                 }
 
                 override fun clicked(event: InputEvent?, x: Float, y: Float) {
-                    val unit = worldScreen.bottomUnitTable.selectedUnit
-                    if (unit == null) return
+                    val unit = worldScreen.bottomUnitTable.selectedUnit ?: return
```
Comment (yairm210) [inline core/src/com/unciv/ui/worldscreen/WorldMapHolder.kt:90]: Also, please stop converting if checks to elvis operators. They're less immediately readable.

File: core/src/com/unciv/ui/utils/ImageGetter.kt (hunk 2 above)
Comment (lishaoxia1985) [reply core/src/com/unciv/ui/utils/ImageGetter.kt:260]: "OtherIcons/TableBackground" is the same as "OtherIcons/whiteDot"

Comment (lishaoxia1985) [reply core/src/com/unciv/ui/utils/ImageGetter.kt:260]: "OtherIcons/TableBackground" is the same as "OtherIcons/whiteDot"

File: core/src/com/unciv/ui/worldscreen/WorldMapHolder.kt (hunk 3 above)
Comment (lishaoxia1985) [reply core/src/com/unciv/ui/worldscreen/WorldMapHolder.kt:90]: But I find some files have used elvis operators.

Comment (yairm210) [reply core/src/com/unciv/ui/worldscreen/WorldMapHolder.kt:90]: Chances are they were also put there by you... 🙃 
Elvis operators do have a limited use when they come as part of a longer chain that would otherwise require fiddly handling 
But for single 'if null return' checks, then 'if null return' is much more readable to the average coder

Comment (lishaoxia1985) [reply core/src/com/unciv/ui/worldscreen/WorldMapHolder.kt]: I reedit it to original branch. In fact, when i find you merge someone's commit i think you change you mind, sorry😢
